messaging/src/main/java/org/axonframework/eventhandling/pooled/WorkPackage.java

```diff
@@ -660,6 +664,19 @@ Builder clock(Clock clock) {
             return this;
         }
 
+        /**
+         * Defines the {@link Runnable} used to clear the cache.
+         *
+         * @param clearCacheFunction the {@link Runnable} used for clearing the cache on the
+         *                           {@link org.axonframework.eventhandling.EventProcessor} for this workpackage
+         *                           segemnt.
+         * @return the current Builder instance, for fluent interfacing
+         */
+        Builder clearCacheFunction(Runnable clearCacheFunction) {
```

Comment:
If we rename the operation on the `EventHandlerInvoker`, I anticipate we rename the builder method too.
Perhaps something like `shutdownAction` (as present in the `Coordinator`) or `abortAction` would make sense.

Reply:
It's now done from the `Coordinator` on releasing the token. So no changes to the `Workpackage` are needed anymore.